[EDU-3889] feat: add guide - new template (#762)
aziontech/docs: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx

</Button>

#### XXXXX

The **XXXXXXX** template contains common caching rules and settings to improve the delivery and performance of static and dynamic content, as well as images, based on the file extension of the requested content. Additionally, it also enables image processing and optimization. This way, you can deliver your content using Azion as a CDN.

<Button href="/en/documentation/products/guides/XXXXXXXX/" text="go to the XXXXXXXX template guide" variant="secondary"> 

</Button>
<br />

> <Button href="#" text="deploy this template" variant="secondary"> 

Reviewer: ```suggestion
<Button href="#" text="deploy this template" variant="secondary"> </Button>
```

</Button>

> #### XXXXX

Author: ```suggestion
#### Dynamic and Static File Optimization
```

> The **XXXXXXX** template contains common caching rules and settings to improve the delivery and performance of static and dynamic content, as well as images, based on the file extension of the requested content. Additionally, it also enables image processing and optimization. This way, you can deliver your content using Azion as a CDN.

Reviewer 2: ```suggestion
The **XXXXXXX** template contains standard cache policies and settings to improve the delivery and performance of static and dynamic content, as well as images, based on the file extension of the requested content. Additionally, it also enables image processing and optimization. This way, you can deliver your content using Azion as a CDN.
```

> The **XXXXXXX** template contains standard cache policies and settings to improve the delivery and performance of static and dynamic content, as well as images, based on the file extension of the requested content. Additionally, it also enables image processing and optimization. This way, you can deliver your content using Azion as a CDN.

Author: ```suggestion
The **Dynamic and Static File Optimization** template contains standard cache policies and settings to improve the delivery and performance of static and dynamic content, as well as images, based on the file extension of the requested content. Additionally, it also enables image processing and optimization. This way, you can deliver your content using Azion as a CDN.
```

> <Button href="/en/documentation/products/guides/XXXXXXXX/" text="go to the XXXXXXXX template guide" variant="secondary"> 

Author: ```suggestion
<Button href="/en/documentation/products/guides/ynamic-and-static-file-optimization-template/" text="go to the Dynamic and Static File Optimization template guide" variant="secondary"> 
```

</Button>
<br />

> <Button href="#" text="deploy this template" variant="secondary"> 

Author: ```suggestion
<Button href="https://manager.azion.com/build-application/build/choose-template/template/azion/dynamic-and-static-file-optimization" text="deploy this template" variant="secondary"> 
```
